google/import_kms_key_ring_test.go

```diff
@@ -0,0 +1,41 @@
+package google
+
+import (
+	"testing"
+
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccGoogleKmsKeyRing_importBasic(t *testing.T) {
+	t.Parallel()
```

Comment:
I'm unsure if I will leave this here if I go down the route of running this test in its own GCP project.